desktop/core/src/desktop/models.py

```diff
@@ -1800,7 +1800,7 @@ def get_config(self):
       ],
       'default_sql_interpreter': default_sql_interpreter,
       'cluster_type': self.cluster_type,
-      'has_computes': self.cluster_type in ('altus', 'snowball'), # or any grouped engine connectors
+      'has_computes': self.cluster_type in ('cdw', 'altus', 'snowball'), # or any grouped engine connectors
```

Comment:
Let's remove altus and snowball? They are not used anywhere right?
@JohanAhlen Any idea?

Reply:
I am thinking of creating a separate commit to remove the altus/snowball related code.